aws/tf/modules/sra/credential.tf

```diff
@@ -50,6 +50,31 @@ resource "aws_iam_role_policy" "cross_account" {
           "*"
         ]
       },
+      {
```

Comment:
Why diverge from the Docs? The docs just have this bundled with the above block

Reply:
personal opinion that they should be split out to make it easier to digest